_posts/2022-02-15-label-noise-intro.md

```diff
@@ -0,0 +1,131 @@
+
+### Confident Learning
+
+Confident Learning high level idea - When the predicted probability of an example is greater than a per-class-threshold, we confidently count that example as actually belonging to that threshold’s class. The thresholds for each class are the average predicted probability of examples in that class. 
+
+Confident Learning estimates a joint distribution between noisy observed labels and the true latent labels. It assumes that the predicted probabilities are out-of-sample holdout probabilities (eg. K-fold cross validation). If this isn't the case then overfitting may occur. Their algorithm also assumes that class-conditional label noise transitions are data independent.
+
+Metrics using a model trained on noisy labels
+
+
+Results are generated using the `get_noise_indices()` method
+
+
+Tested on a separate test set
+
+|                                                                                                                            | precision | recall | f1-score | support |
+|----------------------------------------------------------------------------------------------------------------------------|-----------|--------|----------|---------|
+| wrong_tag_inscope                                                                                                          | 0.22      | 0.59   | 0.32     | 99      |
+| wrong_tag_oos                                                                                                              | 0.55      | 0.63   | 0.59     | 136     |
+| wrong_tags_aoos                                                                                                            | 0.27      | 0.59   | 0.37     | 99      |
+
+Results are slightly better when the model is trained on clean data
+
+|                                                                                                                            | precision | recall | f1-score | support |
+|----------------------------------------------------------------------------------------------------------------------------|-----------|--------|----------|---------|
+| wrong_tag_inscope                                                                                                          | 0.25      | 0.71   | 0.37     | 99      |
+| wrong_tag_oos                                                                                                              | 0.58      | 0.72   | 0.64     | 136     |
+| wrong_tags_aoos                                                                                                            | 0.30      | 0.69   | 0.42     | 99      |
+
+We expect cleanlab to perform even better once our model test accuracies improve. Cleanlab wont be very useful if the model is performing poorly even on a clean dataset.
+
+## Minimizing tagging errors at source
```

Comment:
should we club this with the sources of error section, or atleast keep them close ?